website/project/model.py

```diff
@@ -2173,6 +2186,7 @@ def fork_node(self, auth, title='Fork of '):
         # Clear permissions before adding users
         forked.permissions = {}
         forked.visible_contributor_ids = []
+        forked.save()
```

Comment:
It can't be imported here; I made it a signaled function to avoid a circular import. 

Also, no, but I can make it so by `try/except`ing a `KeyExistsError`.